src/components/WalletConnect/CreateTokenModal.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigation } from '@react-navigation/native';
+import { t } from 'ttag';
+import { StyleSheet, Text } from 'react-native';
+import { COLORS } from '../../styles/themes';
+import { ModalBase } from '../ModalBase';
+import { WarnDisclaimer } from './WarnDisclaimer';
+import { walletConnectReject } from '../../actions';
+
+const styles = StyleSheet.create({
+  body: {
+    paddingBottom: 24,
+  },
+  text: {
+    fontSize: 14,
+    lineHeight: 20,
+  },
+  bold: {
+    fontWeight: 'bold',
+  },
+  selectionContainer: {
+    borderRadius: 8,
+    paddingVertical: 8,
+    paddingHorizontal: 16,
+    backgroundColor: COLORS.freeze100,
+  },
+});
+
+export default ({
+  onDismiss,
+  data,
+}) => {
+  const navigation = useNavigation();
+  const dispatch = useDispatch();
+  const isRetrying = useSelector((state) => state.walletConnect.createToken.retrying);
+
+  // XXX: Make it navigates to readMoreUrl
+  const onReadMore = () => {};
```

Comment:
What is this URL? Have you confirmed with @BrunoCampana the deployment of this doc?